libkbfs/keybase_service_base.go

```diff
@@ -292,7 +292,14 @@ func (k *KeybaseServiceBase) Identify(ctx context.Context, assertion, reason str
 		Reason:        keybase1.IdentifyReason{Reason: reason},
 	}
 	res, err := k.identifyClient.Identify2(ctx, arg)
-	if err != nil {
+	// Identify2 still returns the user-plus-keys data, even if it
```

Comment:
Ah, our messages crossed. Yes, I can clarify the comment.

1) I don't know whether there will be a tracker popup (in my limited testing with the "cboss123" user, I never saw one), but why does it matter?  We're not encrypting for any keys yet.  As soon as we rekey, we'll call `Identify2` again, and that will have a popup (just like sharing before signup, right?).
2) Yep.